src/bootstrap-tenant/bootstrapper/items.ts

```diff
@@ -1229,25 +1238,37 @@ export async function setItems({
       if (item._componentsData?.[language]) {
         Object.keys(item._componentsData[language]).forEach(
           (componentId: string) => {
-            const componentContent: ComponentContentInput =
+            const componentContent: any =
               item._componentsData?.[language][componentId]
 
-            updates.push(() =>
-              context.callPIM(
-                buildUpdateItemComponentQueryAndVariables({
-                  itemId,
-                  language,
-                  input: {
-                    componentId,
-                    ...componentContent,
-                  },
-                })
+            if (
+              // @ts-ignore
```

Comment:
This should be extracted to a function, that takes `componentContent: ComponentContentInput` as input. Make a descriptive name for the function, like `shouldComponentContentBeSet`, or something along those lines. That will allow us to test it in isolation too.

Reply:
I've created a hookd and extended the types, but after realized we don't need it, ive just made changes to the componentInput function where it returns null for itemRelations and then checks if componentContent is truthy before pushing it to updates array.